src/commands/platforms/create.ts

```diff
@@ -138,15 +165,23 @@ export default class PlatformCreate extends BaseCommand {
         message: 'What type of platform would you like to register?',
         choices: [
           'kubernetes',
+          ...(this.app.config.environment !== ENVIRONMENT.PRODUCTION ? ['agent (BETA)'] : []),
         ],
       },
     ]);
+    if (!flags.type && platform_type_answers.platform_type.indexOf('agent') === 0) {
```

Comment:
Rather than searching with `indexOf, why not define the string above the prompt, then just check for equality with that?